README.md

```diff
@@ -25,12 +25,41 @@ Composer will install the library to your project's `vendor/goaop/parser-reflect
 Usage
 ------------
 
+### Initialization
+
+Prior to first use library needs can optionally be initialized.
```

Comment:
Looks like there are typo: library needs can optionally be initialized. Is it correct sentence in English? Please, consider rewriting it.